app/controllers/v1/events_controller.rb

```diff
@@ -9,13 +9,20 @@ def show
   end
 
   def create
-    @event = Event.create(event_params)
-    render json: @event
+    @event = Event.new(event_params)
```

Comment:
I thought about it, but in `save` control flow I usually find the classic `if/else` form. I think that is mainly caused due readability: 
* the classic `if/else` with short calls is really easy to read.
* a guard clause is more like point breaker for required preconditions [avoinding nested conditionals](https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals). I think this is not the case and it would result in a harder to read control flow.

Anyway, if we would move to a guard clause, It would use it tomanage the exception, not the desired behavior:
```ruby 
def create
  @event = Event.new(event_params)
  return 'validation_error', status: :forbidden unless @event.save
  render json: @event
end
```